src/common/KernelBase.hpp

```diff
@@ -256,6 +277,13 @@ class KernelBase
       hipErrchk( hipDeviceSynchronize() );
     }
 #endif
+#if defined(RAJA_ENABLE_SYCL)
+    if ( running_variant == Base_SYCL ||
+         running_variant == RAJA_SYCL ) {
+      getSyclResource().wait();
```

Comment:
Since this wait is here we sould remove the wait in the calling code.

Reply:
Will be fixed in PR that resolves https://github.com/LLNL/RAJAPerf/issues/434